packages/dev/core/src/Inputs/scene.inputManager.ts

```diff
@@ -340,7 +341,7 @@ export class InputManager {
 
     private _processPointerDown(pickResult: Nullable<PickingInfo>, evt: IPointerEvent): void {
         const scene = this._scene;
-        if (pickResult && pickResult.hit && pickResult.pickedMesh) {
+        if (pickResult?.pickedMesh) {

```

Comment:
Why do we not need to check `pickResult.hit` anymore?

Reply:
Since `pickResult.hit` can't be true without a value for `pickResult.pickedMesh`, we only need to check for one of these values, so I just compressed it to check if we have a mesh.  If it's more optimal, I can just change it to check `hit`

Reply:
This code is in a number of places. Might be good to replace them all or just don't change it?

Reply:
Okay, I'll replace all instances in the InputManager to match with this.